xml/System.Activities.Statements/ForEach`1.xml

```diff
@@ -25,7 +25,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- <xref:System.Activities.Statements.ForEach> is similar to the keyword `foreach`, but is implemented as an activity rather than a language statement. Note that unlike `foreach` in C#, the <xref:System.Activities.Statements.ForEach%601> activity uses <xref:System.Collections.IEnumerable%601.GetEnumerator%2A>, rather than <xref:System.Collections.IEnumerator%23System%23Collections%23IEnumerable%23GetEnumerator%2A>.  
+ <xref:System.Activities.Statements.ForEach`1> is similar to the keyword `foreach`, but is implemented as an activity rather than a language statement. Note that unlike `foreach` in C#, the <xref:System.Activities.Statements.ForEach%601> activity uses <xref:System.Collections.IEnumerable.GetEnumerator*>, rather than <xref:System.Collections.IEnumerable.GetEnumerator*>.  
```

Comment:
Yes, it should definitely reference `IEnumerable<T>.GetEnumerator` rather than IEnumerable.GetEnumerator.